src/style.css

```
@@ -0,0 +1,99 @@
+@import './../src/partials/button/button.css';
+@import './../src/partials/input/input.css';
+@import './../src/partials/avatar/avatar.css';
+
+*{
+  font-family: 'Tenor Sans', sans-serif;
+}
+
+body {
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  height: 100vh;
+  width: 100%;
+background-color: #96c8fb;
+background-image: linear-gradient(315deg, #96c8fb 0%, #ddbdfc 74%);
+}
+
+.login-form {
+  &__container{
```

Стоит привести к одному виду так или с пробелом
```suggestion
.login-form{
  &__container{
```